### PR TITLE
NF: use a tuple class instead of int[]

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
This help debugging all tasks passing deck study data between background and foreground, ensuring all data are at the
correct place.

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/StudyOptionsFragment.java [hunk 1]
```diff
@@ -526,30 +526,41 @@ protected void refreshInterface(boolean resetSched, boolean resetDecklist) {
     }
 
 
+    public static class DeckStudyData {
+        public final int mNewCards;
+        public final int mLrnCards;
+        public final int mRevCards;
+        public final int mTotalNew;
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/StudyOptionsFragment.java:533]: Can't think of a better name off the top of my head, but `TotalNew` and `TotalCards` are similar enough to get confused

**Review by david-allison-1 — APPROVED** [review]
LGTM - much cleaner

File: AnkiDroid/src/main/java/com/ichi2/anki/StudyOptionsFragment.java (hunk 1 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/anki/StudyOptionsFragment.java]: Name changed. Javadoc added

**Review by mikehardy — APPROVED** [review]
yes for strong typing - thanks @Arthur-Milchior - concerns David mentioned were addressed, looks good to go, to me

**Pull request merged**